src/utils/wallet.js

```diff
@@ -183,7 +184,7 @@ class Wallet {
                         break
                     }   
                 }
-                this.selectAccount(nextAccountId)
+                store.dispatch(selectAccount(nextAccountId))
```

Comment:
@marcinbodnar do I understand correctly that larger plan is to break down `wallet.js` into Redux actions?

Reply:
@vgrichina Yes, that's the larger plan - in this case, we want to clear the account redux state while changing account.